ra/ra.go

```diff
@@ -2360,8 +2355,11 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 		if existingOrder.Id == 0 || existingOrder.Created == 0 || existingOrder.Status == "" || existingOrder.RegistrationID == 0 || existingOrder.Expires == 0 || len(existingOrder.Names) == 0 {
 			return nil, errIncompleteGRPCResponse
 		}
+		// Track how often we reuse an existing order and how old that order is.
+		ra.orderAges.WithLabelValues("NewOrder").Observe(ra.clk.Since(time.Unix(0, existingOrder.Created)).Seconds())
 		return existingOrder, nil
 	}
+	ra.orderAges.WithLabelValues("NewOrder").Observe(0)
```

Comment:
I wouldn't do this until line 2510, right before we successfully return the new order. Any time before that, this new order might not actually get created.

Reply:
Oh, yep, good call.